pkg/raft/testdata/msg_app_commit_index.txt

```diff
@@ -189,3 +189,9 @@ stabilize 1 2 3
 > 1 receiving messages
   2->1 MsgHeartbeatResp Term:1 Log:0/0
   3->1 MsgHeartbeatResp Term:1 Log:0/0
+
+status 1
+----
+1: StateReplicate match=13 next=14 sentCommit=11 matchCommit=11
```

Comment:
I wonder why the commit haven't converged here, after `stabilize`. Is this because we're the leader, and we don't keep these indices up to date on the leader? Might be worth fixing that for consistency (in another commit, as always :))?

This resembles the inconsistency that we had for `Match/Next`, fixed in #126902.